src/hiero_sdk_python/response_code.py

```
@@ -375,13 +376,13 @@ def is_unknown(self) -> bool:
         return self.name.startswith("UNKNOWN_CODE_")
 
     @classmethod
-    def get_name(cls,code: int) -> str:
+    def get_name(cls, code: int) -> str:
         """
         Returns the name of the response code.
         """
         warnings.warn(
```

Ruff (0.14.14)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)